client_source/sqlplus/ut_run.sql

```diff
@@ -70,7 +85,12 @@ Parameters:
 
 whenever sqlerror exit failure
 whenever oserror exit failure
-conn &1
+
+define client_path="&1"
```

Comment:
We have the same documentation on `ut_run-script.md`, maybe we should remove this and let a link to the one in the userguide. What do you think?